1.3/Defs/Interactions.xml

```diff
@@ -1375,7 +1411,8 @@
 						<li>stressed->*glowers*</li>
 						<li>stressed(INITIATOR_mood&lt;=0.2)->*mutters angrily*</li>
 						
-						<li>calm->*smiles*</li>
+						<li>calm(p=2)->*smiles*</li>
+						<li>calm->Here we are!</li>
```

Comment:
Didn't understand... "here we are"?

Reply:
A filler sentence. Research indicates that this is highly colloquial to my upbringing - my mistake, I thought _everyone_ said "here we are!" to just sort of say "I exist and that's just fantastic" :) Removing.